docs/nginx-reverse-proxy.md

```diff
@@ -0,0 +1,99 @@
+# Nginx Reverse Proxy
+
+As already mentioned, this project relies on [pluswerk/docker-global](https://github.com/pluswerk/docker-global).
+
+Here is an information about the VIRTUAL_* variable clarified.
+
+## VIRTUAL_HOST: Virtual host (your domain)
+
+So that the nginx reverse proxy knows which domain belongs to which Docker container there is the VIRTUAL_HOST variable.
+
+##### Variant A
+
+You can simply specify this domain directly without regular expressions. See variant A.
```

Comment:
```suggestion
You can specify this domain directly without regular expressions. See variant A.
```

As described more clearly here: https://css-tricks.com/words-avoid-educational-writing/
Words like `simply` shouldn't be present in Documentation or any code related writings.